keras_cv/models/object_detection/yolo_v8/yolo_v8_label_encoder.py

```diff
@@ -163,55 +165,74 @@ def call(
                     box should be excluded from both class and box losses.
         """
         if isinstance(gt_bboxes, tf.RaggedTensor):
-            raise ValueError(
-                "`YOLOV8LabelEncoder`'s `call()` method does not "
-                "support RaggedTensor inputs for the `gt_bboxes` argument. "
-                f"Received `type(gt_bboxes)={type(gt_bboxes)}`."
+            dense_bounding_boxes = bounding_box.to_dense(
+                {"boxes": gt_bboxes, "classes": gt_labels},
+            )
+            gt_bboxes = dense_bounding_boxes["boxes"]
+            gt_labels = dense_bounding_boxes["classes"]
+
+        if isinstance(mask_gt, tf.RaggedTensor):
+            mask_gt = mask_gt.to_tensor()
+
+        max_num_boxes = tf.cast(tf.shape(gt_bboxes)[1], dtype=tf.int64)
+
+        def handle_call(
```

Comment:
Maybe let's call this `encode_to_targets`

Reply:
Done